docker/README.md

```diff
@@ -0,0 +1,29 @@
+docker containers for fishtest server (TODO) and workers
+
+
+### local worker
+
+Example of starting a single fishtest worker locally
+
+```bash
+NUM_WORKERS=1 WORKER_ARGS="username password" docker compose up --build -d
```

Comment:
This syntax isnt really os independent. I think it should use https://docs.docker.com/build/guide/build-args/.

Reply:
using .env resolves defining env vars independently of os. when using .env, we can skip using build args rather than taking an extra hop: `host env -> build args -> container env`. the docs also suggest to not use build args for credentials:

https://docs.docker.com/engine/reference/builder/#arg

<img width="801" alt="image" src="https://github.com/official-stockfish/fishtest/assets/208617/75233e44-0e9a-4626-9581-2602987994a4">

Reply:
opting to use env vars directly instead of intermediate build args, since this avoids having to rebuild the image every time .env changes. with env vars, the container can be restarted to pick up changes. also addresses the big red warning from docker docs